### PR TITLE
Bump Arktos to v1.0

### PR DESCRIPTION
**Pull request by Sindica**
Also update mizar to stable release version v0.9

### CODE REVIEW

File: hack/arktos-up.sh [hunk 1]
```diff
@@ -556,8 +556,7 @@ fi
 # Applying mizar cni
 if [[ "${CNIPLUGIN}" == "mizar" ]]; then
   ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" create configmap system-source --namespace=kube-system --from-literal=name=arktos --from-literal=company=futurewei
-  # ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" apply -f ${KUBE_ROOT}/hack/testdata/mizar/deploy.mizar.next.yaml 
-  ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" apply -f https://raw.githubusercontent.com/CentaurusInfra/mizar/dev-next/etc/deploy/deploy.mizar.dev.yaml
+  ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" apply -f https://raw.githubusercontent.com/CentaurusInfra/mizar/v${MIZAR_VERSION}/etc/deploy/deploy.mizar.yaml
```
[the remark this answers is not in the document]
Comment (Sindica) [reply hack/arktos-up.sh:559]: > This should change to download from mizar official release version: [https://github.com/CentaurusInfra/mizar/releases/download/v${NETWORK_PROVIDER_VERSION}/deploy.mizar.yaml](https://github.com/CentaurusInfra/mizar/releases/download/v$%7BNETWORK_PROVIDER_VERSION%7D/deploy.mizar.yaml)

Local dev env use different different variable name for Mizar version.

[the remark this answers is not in the document]
Comment (Sindica) [reply hack/arktos-up.sh]: > This should change to download from mizar official release version: [https://github.com/CentaurusInfra/mizar/releases/download/v${NETWORK_PROVIDER_VERSION}/deploy.mizar.yaml](https://github.com/CentaurusInfra/mizar/releases/download/v$%7BNETWORK_PROVIDER_VERSION%7D/deploy.mizar.yaml)

Updated.

**Review by sonyafenge — APPROVED** [review]
/lgtm

**Pull request merged**